.github/ISSUE_TEMPLATE/new-paper-template.md

```diff
@@ -36,6 +42,11 @@ Citation:
 - [ ] biostatistics
 - [ ] immunology
 - [ ] pharmacology
+- [ ] other:
+
+<!--Please fill out as much of the above as you feel able, then submit this issue using the green button below. -->
+# Summary
+If you would like to submit a summary of the paper, please copy and paste the following into a comment.
```

Comment:
Did you want to keep this? I couldn't tell how the correspondence with @LucyMcGowan turned out on it.

Reply:
Corresponded with @rando2 - answer is Yes!